notebooks/224-distilbert-Sequence-Classification/224-distilbert-Sequence-Classification-OV2.0.ipynb

```diff
@@ -0,0 +1,270 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "abc41ac0",
+   "metadata": {},
+   "source": [
+    "# Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fe80a355",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from transformers import AutoModelForSequenceClassification, AutoTokenizer\n",
+    "import openvino.runtime as ov\n",
+    "import warnings\n",
+    "from pathlib import Path\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4dde5b43",
+   "metadata": {},
+   "source": [
+    "# Transformers Serialization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1742fd4b",
+   "metadata": {},
+   "source": [
+    "Check out this link [https://huggingface.co/docs/transformers/serialization] to learn how to convert transformers to onnx format"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5db803ea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -m transformers.onnx -h"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4794f066",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -m transformers.onnx -m distilbert-base-uncased-finetuned-sst-2-english --feature sequence-classification model/"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "782bbebf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "checkpoint = \"distilbert-base-uncased-finetuned-sst-2-english\"\n",
+    "tokenizer = AutoTokenizer.from_pretrained(pretrained_model_name_or_path=checkpoint)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8fff79fc",
+   "metadata": {},
+   "source": [
+    "# Model Optimizer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a0f48c1b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "onnx_model_path = 'model.onnx'\n",
+    "MODEL_DIR = f\"model/\"\n",
+    "onnx_model_path = Path(MODEL_DIR) / onnx_model_path\n",
+    "\n",
+    "!mo --input_model $onnx_model_path --output_dir $MODEL_DIR --model_name $checkpoint --input input_ids,attention_mask --input_shape [1,128],[1,128]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "27cc074e",
+   "metadata": {},
+   "source": [
+    "Creating the input and output features for the inference"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e31a2644",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "warnings.filterwarnings(\"ignore\")\n",
+    "core = ov.Core()\n",
+    "ir_model_xml = str((Path(MODEL_DIR)/checkpoint).with_suffix(\".xml\"))\n",
+    "compiled_model = core.compile_model(ir_model_xml, \"AUTO\")\n",
```

Comment:
for AUTO device assignment no need specify device, this option selected by default